habit-backend/users/admin.py

```diff
@@ -0,0 +1,40 @@
+from .forms import UserAdminCreationForm, UserAdminChangeForm
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.forms import ReadOnlyPasswordHashField
+
+from users.models import User
+
+class UserAdmin(BaseUserAdmin):
```

Comment:
What will be the ID of the user?
Can we have a numeric ID for the users?

Reply:
Yes By default id column is managed by Django internally for all tables , it is numeric and a primary key to a table.